veer/src/main/java/frc/robot/control/ManualInputInterfaces.java

```diff
@@ -0,0 +1,165 @@
+   * @return - a double value associated with the magnitude of the x componet
+   */
+  public double getInputArcadeDriveY()
+  {
+    // need to invert the y for all xbox controllers due to xbox controler having up as negative y axis
+    return driverController.getLeftY() * -1.0;
+  }
+
+  /**
+   * A method to get the spin drive X componet being input from humans
+   * @return - a double value associated with the magnitude of the x componet
+   */
+  public double getInputSpinDriveX()
+  {
+    return driverController.getRightX();
+  }
+
+  /**
+   * A method to get the co-driver 'intake speed' input from humans
+   * @return - a double value associated with the magnitude of left trigger 
+   */
+  public double getBallMotorSpeed()
+  {
+    double targetMotorSpeed = 0.0;
+    double intakeValue = coDriverController.getLeftTriggerAxis();
+    double layupValue = coDriverController.getRightTriggerAxis();
+    if(intakeValue > layupValue && intakeValue > Constants.BallHandlerMotorMinimumAbsoluteInputValue){
+      targetMotorSpeed = intakeValue * Constants.BallHandlerMotorStorageDirectionMultiplier;
+    }
+    else if(layupValue > Constants.BallHandlerMotorMinimumAbsoluteInputValue){
+      targetMotorSpeed = layupValue * Constants.BallHandlerMotorRetrievalDirectionMultiplier;
+    }
+    return targetMotorSpeed;
+  }
+
+  /**
+   * A method to initialize various commands to the numerous buttons.
+   * Need delayed bindings as some subsystems during testing won't always be there.
+   */
+  public void initializeButtonCommandBindings()
+  {
+    // Configure the driver xbox controller bindings
+    if(InstalledHardware.driverXboxControllerInstalled)
+    {
+      this.bindCommandsToDriverXboxButtons();
+    }
+
+    // Configure the co-driver xbox controller bindings
+    if(InstalledHardware.coDriverXboxControllerInstalled)
+    {
+      this.bindCommandsToCoDriverXboxButtons();
+    }
+  }
+
+  /**
+   * Will attach commands to the Driver XBox buttons 
+   */
+  private void bindCommandsToDriverXboxButtons()
+  {
+    if(InstalledHardware.driverXboxControllerInstalled)
+    {
+      // TODO - do we need anything here?
+    }
+  }
+
+  /**
+   * Will attach commands to the Co Driver XBox buttons 
+   */
+  private void bindCommandsToCoDriverXboxButtons()
+  {
+    if(InstalledHardware.coDriverXboxControllerInstalled)
+    {
+      JoystickButton bumperLeft = new JoystickButton(coDriverController, XboxController.Button.kLeftBumper.value);
+      JoystickButton bumperRight = new JoystickButton(coDriverController, XboxController.Button.kRightBumper.value);
+      JoystickButton buttonY = new JoystickButton(coDriverController, XboxController.Button.kY.value);
+      JoystickButton buttonA = new JoystickButton(coDriverController, XboxController.Button.kA.value);
+      JoystickButton buttonB = new JoystickButton(coDriverController, XboxController.Button.kB.value);
+
+      if(subsystemCollection.getBallHandlerSubsystem() != null)
```

Comment:
I don't think these button bindings are the same as what's in the spec: https://docs.google.com/document/d/1-NSeH8xcZ0r-yd30U5SVGjjtIqV9k0ZpX22TS0DgCW4/edit#bookmark=id.3qmyxbm37yjp

Reply:
I updated the spec a bit to add detail but happy to take out anything folks feel is not beneficial.

Reply:
Along these lines, the original spec had mentioned up/down and in/out.  I had originally taken that to imply single actuator (for the up/down) and single motor (for the in/out).  But then on re-read this morning, I thought well maybe there are two pneumatic actuators controlling two axis of movement.  Along with that, I was thinking, well maybe the spec originally missed the ball motor control (as in it had only captured the pneumatic movements).  So I asked a question to the team in email.